code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -163,16 +163,16 @@
 			to_chat(H, "<span class='warning'>Alert: Internal temperature regulation systems offline; thermal damage sustained. Shutdown imminent.</span>")
 			H.visible_message("[H]'s cooling system fans stutter and stall. There is a faint, yet rapid beeping coming from inside their chassis.")
 
+
 /datum/species/ipc/spec_revival(mob/living/carbon/human/H)
 	H.dna.features["ipc_screen"] = "BSOD"
 	H.update_body()
 	H.say("Reactivating [pick("core systems", "central subroutines", "key functions")]...")
-	sleep(3 SECONDS)
-	H.say("Reinitializing [pick("personality matrix", "behavior logic", "morality subsystems")]...")
-	sleep(3 SECONDS)
-	H.say("Finalizing setup...")
-	sleep(3 SECONDS)
+	addtimer(CALLBACK(src, .proc/post_revival, H), 6 SECONDS)
+
+/datum/species/ipc/proc/post_revival(mob/living/carbon/human/H)
+	if(H.stat < DEAD)
+		return
```

Comment:
Is this even possible for a post_revival proc call?

Reply:
This is a proc defined for the first time on this, which is called by a timer 3(?) seconds after spec_revival(). This means that (although unlikely) there is a chance the mob dies in those three seconds and says this while dead.

Reply:
I can and will intentionally trigger this repeatedly ingame.